builds/jenkins-build-pack-website/Jenkinsfile

```diff
@@ -128,8 +128,8 @@ node()  {
 				}
 
 				create_cloudfront_url = true
-				if (service_config['create_cloudfront_url'] == true || service_config['create_cloudfront_url'] == "true") {
-					create_cloudfront_url = true
+				if (service_config['deployment_targets'] && service_config['deployment_targets'] instanceof Object && service_config['deployment_targets'].website == "aws_s3") {
```

Comment:
Changes are good, let's target these for next release instead of targeting `apigee` release?